config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```diff
@@ -356,6 +356,44 @@ presubmits:
             limits:
               cpu: "7"
               memory: "10Gi"
+  - name: pull-kueue-test-e2e-certmanager-main
+    cluster: eks-prow-build-cluster
+    branches:
+      - ^main
+    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$|^(keps|site|charts)/"
+    decorate: true
+    path_alias: sigs.k8s.io/kueue
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: pull-kueue-test-e2e-certmanager-main
+      description: "Run cert-manager end to end tests"
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250227-3a13bdd784-master
+          env:
+            - name: E2E_KIND_VERSION
+              value: kindest/node:v1.31.1
```

Comment:
I think this should be 1.32.

Reply:
I think 1.31.1 is fine for this PR for consistency with other CI jobs, like pull-kueue-test-e2e-tas-main. I think we can have an orthogonal follow up bumping to 1.32.x (basically latest)